Tests/CartonCommandTests/SDKCommandTests.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Carton contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Cavelle Benjamin on Dec/25/20.
+//
+
+@testable import CartonCLI
+import TSCBasic
+import XCTest
+
+extension SDKCommandTests: Testable {}
+
+final class SDKCommandTests: XCTestCase {
+  func testInstall() throws {
+    // given I've created a directory
+    // let package = "Milk"
+    // let packageDirectory = testFixturesDirectory.appending(component: package)
+
+    // let bundle = "Bundle"
+    // let bundleDirectory = packageDirectory.appending(component: bundle)
+
+    // it's ok if there is nothing to delete
+    // do { try bundleDirectory.delete() } catch {}
+
+    // XCTAssertFalse(bundleDirectory.exists, "The Bundle directory should not exist")
```

Comment:
Thanks for setting up the test! Can this commented out code be removed, or is this still relevant?

Reply:
Yes. I wanted to make sure the test passed before deleting anything. So I think we're good to finish all of them after this merge.